solutions/import-workspace/locals_vpc.tf

```diff
@@ -0,0 +1,40 @@
+locals {
```

Comment:
Add block comments:
####################################################
 Locals for outputs
####################################################